app/models/tenant.rb

```diff
@@ -6,5 +6,16 @@ class Tenant < ApplicationRecord
   def self.tenancy_enabled?
     ENV["BYPASS_TENANCY"].blank?
   end
-end
 
+  # as_json serialization for internal API's tenants_controller
+  # overrides Insights::API::Common::OpenApi::Serializer
+  def _schema(arg)
```

Comment:
Is this only needed for the generator? If so - lets remove it since the generator doesn't work for this and we're going to not use the generator anymore.